### PR TITLE
MHV-55332 MHV-58203 Update-Content-And-Left-Navigation-For-Labs-Chem/Hem

### PR DESCRIPTION
**Pull request by tynyttie**
## Summary
This ticket combines the tasks from MHV-55332-Update-Content-And-Left-Navigation-For-Labs-Chem and MHV-58203-Change-word-on-all-print-download. After discussing with UCD, I added "Lab and test results" information to the left side of the navigation bar on the landing page. Additionally, on the VitalDetail page, the download drop-down list items were rephrased by replacing "page" with "list."

Note: It was decided that in ticket MHV-55332, the addition of the settings button to the left navigation will be handled in a new ticket.

## Related issue(s)

[MHV-55332](https://jira.devops.va.gov/browse/MHV-55332) - Update Content and Left Navigation for Labs Chem/Hem
[MHV-58203](https://jira.devops.va.gov/browse/MHV-58203)- Change the wording on all print/download buttons on vitals lists

MHV-55332 - Update Content and Left Navigation for Health Conditions-see latest designs 

https://www.figma.com/design/mGZRdLypKGaFsHo5xp2kaZ/Medical-Records?node-id=4467-36464&t=NouvL14iCrDlRZ5S-0 

From Amelia:  We will need to include the settings page and download all records page. Content for those is most likely changing, too–hopefully we will have those change asap.  (Do we need a separate ticket?)

User Story: As a Medical Records User, I want to see updated content and left navigation so that the latest designs are displayed.

Feature Flag Y/N ? N

Manual Testing Y/N ? Y

Automated Testing Y/N ? N

Accessibility Testing Y/N ? N

UCD Validation Y/N Y

Acceptance Criteria:

AC1  Left Navigation has been updated to latest design

AC2 Content has been updated to latest design

AC3 Verify with UCD 
AC4
AC5

Links to UCD:

Figma Link: https://www.figma.com/design/mGZRdLypKGaFsHo5xp2kaZ/Medical-Records?node-id=4467-36464&t=NouvL14iCrDlRZ5S-0

User Flow:

Mobile:

Desktop:

Other Design Notes:

Architectural Notes: Always check for the latest designs in Figma

MHV-58203 - Change the wording on all print/download buttons on vitals lists to say "Print this list" and "download a PD of this list" and "Download a text file (.txt) of this list".

From Amelia:  

User Story:  As a MR user, I want to 

Feature Flag Y/N ? N

DataDog/Analytics?  Y/N ?

Manual Testing Y/N ? Y-Maruf

Automated Testing Y/N ?

Accessibility Testing Y/N ? Y-Bobby

UCD Validation Y/N Y

Acceptance Criteria:
AC1  
AC2  
AC3 UCD has validated changes (Amelia)
AC4

Links to UCD:
Figma Link:   
Other Design Notes

Architectural Notes:

## Testing done
Conducted UI testing to ensure the expected behavior. I updated existing unit test for new components.

## Screenshots

Screenshot 2024-05-20 at 4 29 37 PM (2)
Screenshot 2024-05-20 at 4 30 05 PM (2)
Screenshot 2024-05-21 at 1 38 01 PM (2)

## What areas of the site does it impact?

The modification affects the Landings page and VitalDetail page of the Medical Records pages.

## Acceptance criteria
"Lab and test results" information to the left side of the navigation bar on the landing page
VitalDetail page, the download drop-down list items were rephrased by replacing "page" with "list."

### Quality Assurance & Testing

- [x] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [x] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [ ] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Authentication

- [x] Did you login to a local build and verify all authenticated routes work as expected with a test user

### CODE REVIEW

**Review by rmessina1010 — DISMISSED** [review]
LGTM

**Review by mmoyer-va — CHANGES_REQUESTED** [review]
In the following bit of code, do the following two things:
1. Add the two missing feature flag conditions for the `<li>` for conditions and labs & tests
2. This whole `<section>` should not be displayed UNLESS any of those `<li>` feature flags are DISABLED.
```
      <section>
        <h2 className="vads-u-margin-top--4 vads-u-margin-bottom--1">
          How to find your other medical records
        </h2>
        <p className="vads-u-margin-bottom--2">
          Right now, only some of your medical records are available here on
          VA.gov. Soon, you’ll be able to find these types of medical records on
          this page:
        </p>
        <ul>
          <li>Lab and test results</li>
          {!displayNotes && <li>Care summaries and notes</li>}
          {!displayVaccines && <li>Vaccines</li>}
          <li>Health conditions</li>
          {!displayVitals && <li>Vitals</li>}
```

File: src/applications/mhv-medical-records/containers/LandingPage.jsx [hunk 1]
```diff
@@ -255,14 +261,11 @@ const LandingPage = () => {
...
-              on VA.gov. And your records on VA.gov only include health
-              information your VA providers have entered.
+              Your medical records on VA.gov only include health information
+              your Va providers have entered.
             </p>
             <p className="vads-u-margin-bottom--2">
-              To find other types of medical records
-              <code>&#8212;</code>
-              including health information you entered yourself
+              To find health information you entered yourself,
```
Comment (mmoyer-va) [inline src/applications/mhv-medical-records/containers/LandingPage.jsx:268]: These should be subject to the same condition as the "Other Medical Records" section above.

**Review by mmoyer-va — CHANGES_REQUESTED** [review]

File: src/applications/mhv-medical-records/containers/LandingPage.jsx (hunk 1 above)
Comment (tynyttie) [reply src/applications/mhv-medical-records/containers/LandingPage.jsx]: fixed.

**Review by mmoyer-va — APPROVED** [review]
Looks good!

**Pull request merged**